### ISSUE TITLE
ckeygen man page has wrong summary and synopsis

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4738]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4738|
|Type|defect|
|Created|2010-11-15 00:36:57Z|
|Branch|https://github.com/twisted/twisted/tree/ckeygen-manpage-4738|

```
ckeygen - connect to SSH servers

SYNOPSIS
       ckeygen [-b bits] [-f filename] [-t type] [-C comment] [-N new passphrase] [-P old passphrase] [-l] [-p] [-q] [-y] ckeygen --help
```

`ckeygen` does not connect to SSH servers.  Also, an extra "ckeygen" near the end is not a required part of the ckeygen command line.

<details><summary>Searchable metadata</summary>

```
trac-id__4738 4738
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_ckeygen_manpage_4738 branches/ckeygen-manpage-4738
branch_author__cyli cyli
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__documentation__easy documentation, easy
time__1289781417000000 1289781417000000
changetime__1295215637000000 1295215637000000
version__None None
owner__cyli cyli
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_1"></a>|@cyli commented|
|-|-|

(In [30442]) Branching to 'ckeygen-manpage-4738'

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_2"></a>|@cyli commented|
|-|-|

(In [30443]) Fixed the ckeygen manpage.

refs [#4738](https://github.com/twisted/twisted/issues/4738)

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_3"></a>|@cyli removed owner|
|-|-|

[Build Results](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/ckeygen-manpage-4738)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra set owner to @cyli|
|-|-|

The man page needs a copyright update:
```
Copyright \(co 2002-2008 Twisted Matrix Laboratories.
```

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_5"></a>|@cyli commented|
|-|-|

(In [30452]) Updated copyright

refs [#4738](https://github.com/twisted/twisted/issues/4738)

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)|@cyli removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun set owner to @cyli|
|-|-|

Looks good, please merge.  Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_8"></a>|@cyli set status to `closed`|
|-|-|

(In [30478]) Merge branch ckeygen-manpage-4738: Fix ckeygen manpage

Author: cyli
Reviewer: exarkun
Fixes: [#4738](https://github.com/twisted/twisted/issues/4738)

Fix the ckeygen manpage to reflect what ckeygen actually does.
